packages/core/src/payment/payment-method-action-creator.ts

```diff
@@ -21,6 +21,47 @@ const APPLEPAYID = 'applepay';
 export default class PaymentMethodActionCreator {
     constructor(private _requestSender: PaymentMethodRequestSender) {}
 
+    loadPaymentMethodsById(
+        methodIds: string[],
+        options?: RequestOptions,
+    ): ThunkAction<LoadPaymentMethodsAction, InternalCheckoutSelectors> {
+        return (store) =>
+            new Observable((observer: Observer<LoadPaymentMethodsAction>) => {
+                const state = store.getState();
+                const cartId = state.cart.getCart()?.id;
+                const params = cartId ? { ...options?.params, cartId } : { ...options?.params };
+
+                observer.next(createAction(PaymentMethodActionType.LoadPaymentMethodsRequested));
+                Promise.all(
+                    methodIds.map(async (id) => {
+                        const response = await this._requestSender.loadPaymentMethod(id, {
+                            ...options,
+                            params,
+                        });
+
+                        return response.body;
+                    }),
+                )
+                    .then((response) => {
+                        observer.next(
+                            createAction(
+                                PaymentMethodActionType.LoadPaymentMethodsSucceeded,
+                                response,
+                            ),
+                        );
+                        observer.complete();
+                    })
+                    .catch((response) => {
+                        observer.error(
+                            createErrorAction(
+                                PaymentMethodActionType.LoadPaymentMethodsFailed,
+                                response,
+                            ),
```

Comment:
We just noticed this error handling will change the current UX when loading a payment method fails.
It appears that when one method fails, all methods cannot work properly, right?

Reply:
Yes, for this when one fails all might need to be failed. Isn't the experience similar right now?

Reply:
The current implementation handles each failure separately and adjusts the layout accordingly.

Reply:
I think this should work similar way, since we still have the fallback in each strategy so that effectively works as a retry.